corehq/apps/styleguide/templates/styleguide/bootstrap5/atoms/accessibility.html

```diff
@@ -108,50 +101,139 @@ <h5>A11y check</h5>
-      also a good indicator if it will be visible to a screen reader. Read more about potential barriers to effective
-      keyboard navigation and keyboard testing <a href="https://webaim.org/techniques/keyboard/">here</a>.
-    </li>
-    <li>
-      Explore the accessibility tree: An accessibility tree is a part of your website's DOM. All
-      accessible objects are created in the accessibility tree for each DOM element that needs to be exposed to
-      assistive technologies. From it, you can see how the page elements are included and organized. To view the
-      accessibility tree of a page under development, we can use
-      <a href="https://developers.google.com/web/tools/chrome-devtools/accessibility/reference">Chrome DevTools' accessibility feature</a>.
-    </li>
-    <li>
-      <em>*this is by no means a complete list and covers only a portion of what makes a site accessible to all users</em>
-    </li>
-  </ul>
+  <h5>tabindex Attribute</h5>
+    <p>The tabindex attribute specifies the tab order of an element and determines whether an element can receive
+      keyboard focus. Here are some scenarios where you might use tabindex:</p>
+    <ul>
+        <li>Focusable but Not Natively Focusable Elements: Some elements like &lt;div&gt; or &lt;span&gt; are not
+          natively focusable, but you might want them to receive focus for user interaction. In such cases, you can
+          add tabindex="0" to make them focusable. Note: we avoid using custom focus order using tab index with values
+          above 0. Instead, make sure DOM elements are ordered correctly for navigability, then modify visual
+          positioning with CSS if needed.</li>
+        <li>Remove Focusability: to explicitly remove an item</li>
+        <li>Dynamic Focus Control: If you have dynamically generated or hidden elements that need to become focusable
+          under certain conditions, you can dynamically add or remove the tabindex attribute.</li>
+    </ul>
+    <h5>aria-hidden Attribute</h5>
+    <p>The aria-hidden attribute indicates whether an element is visible or hidden to assistive technologies. It does
+      not affect the visual rendering of the page. Here are some scenarios where you might use aria-hidden:</p>
+    <ul>
+        <li>Hiding Decorative Elements: If you have decorative or presentational elements that should be hidden from
+          screen readers, you can use aria-hidden="true".</li>
+        <li>Dynamic Content Updates: When dynamically updating content, you may want to temporarily hide or show
+          certain elements from assistive technologies without altering their visual display.</li>
+        <li>Modal Dialogs or Popovers: When displaying modal dialogs or popovers, you may want to hide the underlying
+          content from screen readers to prevent confusion.</li>
+    </ul>
+    <h5>When to Use Each</h5>
+    <p>Use tabindex when you need to control the tab order or make non-focusable elements focusable.</p>
```

Comment:
> `control the tab order`

I think this is misleading based on `avoid using custom focus order using tab index with values above 0` further up. Maybe something like:
```suggestion
    <p>Use tabindex when you need to remove or add tab focusability to an element.</p>
```